ui/src/main/java/com/ruslan/controller/OrderController.java

```diff
@@ -0,0 +1,97 @@
+package com.ruslan.controller;
+
+import com.ruslan.controller.webExceptions.NoSuchEntityException;
+import com.ruslan.dto.OrderDto;
+import com.ruslan.entity.order.OrderStatus;
+import com.ruslan.services.OrderService;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@Log4j2
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+
+    private final OrderService orderService;
+
+    @Autowired
+    public OrderController(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    @GetMapping("/all")
+    public ResponseEntity<List<OrderDto>> showAllOrders() {
+        log.info("Received GET request /orders");
+        return ResponseEntity.ok(orderService.listOrderDto());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<OrderDto> showOrder(@PathVariable int id) {
+        log.info("Received GET request /orders/" + id);
+        return ResponseEntity.ok(orderService.findById(id));
+    }
+
+    @PostMapping("/")
+    public ResponseEntity<OrderDto> saveOrder(@RequestBody OrderDto dto) {
+        log.info("Received POST request /orders");
+        orderService.saveOrder(dto);
+        return ResponseEntity.ok(dto);
+    }
+
+    @PutMapping("/")
+    public ResponseEntity<OrderDto> updateOrder(@RequestBody OrderDto dto) {
+        log.info("Received PUT request /orders");
+        orderService.saveOrder(dto);
+        return ResponseEntity.ok(dto);
+    }
+
+    @DeleteMapping("/{id}")
+    public String deleteOrder(@PathVariable int id) {
```

Comment:
Better have generic response dto with string value inside it (or Map atleast) then just string, in this way you always will have json as response